src/main/java/uk/gov/cshr/civilservant/repository/QuizRepository.java

```diff
@@ -1,15 +1,28 @@
 package uk.gov.cshr.civilservant.repository;
 
+import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+import org.springframework.security.access.prepost.PreAuthorize;
 import org.springframework.stereotype.Repository;
+import uk.gov.cshr.civilservant.domain.AllCivilServantDetails;
 import uk.gov.cshr.civilservant.domain.Quiz;
 
+import java.util.List;
 import java.util.Optional;
 
 @Repository
-public interface QuizRepository extends CrudRepository<Quiz, Long> {
+//@PreAuthorize("isAuthenticated()")
+@RepositoryRestResource
```

Comment:
I know a few other repositories in csrs use `@ RepositoryRestResource` but I don't think we're actively going to use it if we're putting rest logic in the controllers. Can we remove?